tests/playwright/src/model/pages/cli-tools-page.ts

```diff
@@ -34,5 +37,54 @@ export class CLIToolsPage extends SettingsPage {
     this.heading = this.header.getByRole('heading', { name: 'CLI Tools', exact: true });
     this.content = this.main.getByRole('region', { name: 'Content' });
     this.toolsTable = this.content.getByRole('table', { name: 'cli-tools' });
+    this.dropDownDialog = page.getByRole('dialog', {
+      name: 'drop-down-dialog',
+    });
+    this.versionInputField = this.dropDownDialog.getByRole('textbox');
+  }
+
+  public getToolRow(toolName: string): Locator {
+    return this.toolsTable.getByRole('row', { name: toolName, exact: true });
+  }
+
+  public getInstallButton(toolName: string): Locator {
+    return this.getToolRow(toolName).getByLabel('Install', { exact: true });
+  }
+
+  public getUninstallButton(toolName: string): Locator {
+    return this.getToolRow(toolName).getByLabel('Uninstall', { exact: true });
+  }
+
+  public getUpdateButton(toolName: string): Locator {
+    return this.getToolRow(toolName).getByLabel('Update', { exact: true });
+  }
+
+  public getVersionSelectionButton(version: string): Locator {
+    return this.dropDownDialog.getByRole('button', { name: version });
+  }
+
+  public async getCurrentToolVersion(toolName: string): Promise<string> {
+    if ((await this.getToolRow(toolName).getByLabel('no-cli-version', { exact: true }).count()) > 0) {
+      return '';
```

Comment:
Is empty string safe?

Reply:
certainly, it permits validating against tobeTruthy() and toBeFalsy()